Marlin/Configuration_adv.h

```diff
@@ -482,12 +482,19 @@
 // if unwanted behavior is observed on a user's machine when running at very slow speeds.
 #define MINIMUM_PLANNER_SPEED 0.05 // (mm/s)
 
-//
-// Use Junction Deviation instead of traditional Jerk Limiting
-//
-//#define JUNCTION_DEVIATION
-#if ENABLED(JUNCTION_DEVIATION)
-  #define JUNCTION_DEVIATION_MM 0.02  // (mm) Distance from real junction edge
+/**
+ * Default Jerk (mm/s)
+ * Override with M205 X Y Z E
+ *
+ * "Jerk" specifies the minimum speed change that requires acceleration.
+ * When changing speed and direction, if the difference is less than the
+ * value set here, it may happen instantaneously.
+ */
+#if DISABLED(JUNCTION_DEVIATION)
+  #define DEFAULT_XJERK               10.0
+  #define DEFAULT_YJERK               10.0
+  #define DEFAULT_ZJERK                0.3
+  #define DEFAULT_EJERK                5.0
```

Comment:
`DEFAULT_EJERK` is still used in the case where `JUNCTION_DEVIATION` and `LIN_ADVANCE` are both enabled.

Reply:
so it was not a good idea?

Reply:
I do like the idea of making junction deviation more up-front. It is now as important as the old jerk settings. So I'll add a commit shortly moving things around a bit….